ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Operation/10_ManageReceipts.md

[!!Documents](RetailSuiteDocorg)  
[!!Einkauf](RetailSuiteSourcing)   
[!!Zahlungsabwicklung](RetailSuiteBanking)  
[!!User Interface bookings](../UserInterface/01a_Bookings.md)  
[!!User Interface book receipts](../UserInterface/01f_BookReceipts.md)  
[!!User Interface supplier receipts](../UserInterface/01_Book.md#supplier-receipts)  
[!!User Interface debtors/creditors](../UserInterface/02a_DebtorsCreditors.md)    

# Manage receipts

Receipts are used to document all transactions and, therefore, play an indispensable role for a correct accounting. There are many different types of receipts, such as invoice, order confirmation, or credit note, among others.

The *Accounting* module offers three possible ways to manage and assign receipts to bookings:

> - Clicking the [ASSIGN RECEIPT] button in the *BOOKINGS* tab to assign a receipt to an already existing booking in the bookings list; see [Retrieve a receipt](#retrieve-a-receipt).

Reviewer: ```suggestion
- Clicking the [ASSIGN RECEIPT] button in the *POSTINGS* tab to assign a receipt to an existing posting in the postings list; see [Retrieve a receipt](#retrieve-a-receipt).
```

> - Clicking the [ASSIGN RECEIPT] button in the *BOOKINGS* tab to assign a receipt to an already existing booking in the bookings list; see [Retrieve a receipt](#retrieve-a-receipt).
> - Clicking the *BOOK RECEIPTS* tab to assign a receipt available as a PDF and create a booking manually; see [Book a receipt](#book-a-receipt).
> - Clicking the *Supplier receipts* menu entry in the *EDIT* context menu to specifically assign a supplier receipt; see [Book a supplier receipt](#book-a-supplier-receipt)

Reviewer: ```suggestion
- Click the [ASSIGN RECEIPT] button in the *POSTINGS* tab to assign a receipt to an existing posting in the postings list; see [Retrieve a receipt](#retrieve-a-receipt).
- Access the *POST RECEIPTS* tab to assign a receipt available as a PDF and create a posting manually; see [Post a receipt](#post-a-receipt).
- Click the *Supplier receipts* menu entry in the *EDIT* context menu to specifically assign a supplier receipt; see [Post a supplier receipt](#post-a-supplier-receipt)
```

> [Info] *Supplier receipts* is a special feature that interacts with the *Purchasing* and *Payment processing* modules, and therefore the appropriate settings must be configured first.

> When a receipt is booked in the *BOOK RECEIPTS* tab, it is linked to the assigned booking. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon will be then displayed next to the booking in the bookings list. When clicking the ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon, the corresponding receipt is displayed.

Reviewer: Ich glaube die Info kannst du hier löschen - die erwähnst du genau richtig an den entsprechenden Stellen in der Procedure.

[comment]: <> (Work in progress!)

## Retrieve a receipt

> The [ASSIGN RECEIPT] button in the *BOOKINGS* tab allows to retrieve and assign a receipt that has not been registered yet to an existing booking. Any unregistered receipts uploaded in the system are displayed in the *BOOK RECEIPTS* tab.  

Reviewer: ```suggestion
You can retrieve and assign a receipt that has not yet been registered to an existing posting. Any unregistered receipts uploaded in the system are displayed in the *POST RECEIPTS* tab.  
```

After being uploaded, the receipts are stored in the *New documents* folder in the *Documents* module. Once they are booked, they are moved automatically into the *Registered documents* folder and then placed in the appropriate subfolder according to the following structure:

- 10000-69999 personal account numbers in case of a debtors'/customers' document
- 70000-79999 personal account numbers in case of a creditors'/suppliers' document

#### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).

> - A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).

Author: ```suggestion
- A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).
```

- A manual booking is created, see [Create a manual booking](./04_CreateManualBooking.md).

> - A receipt is uploaded in PDF format in the *Documents* module.

Reviewer: ```suggestion
- A receipt is uploaded as a PDF in the *Documents* module.
```
PDF = portable document format
Ist PDF das einzige unterstütze Format?

#### Procedure

*Accounting > Select fiscal year > Book > Tab BOOKINGS*

> ![Assign receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AssignReceipt.png "[Assign receipt]")

Reviewer: Bild ändern -> keine Buchung ausgewählt

1. Select the booking to be assigned a receipt from the bookings list in the *BOOKINGS* tab.  
The existing booking details are displayed in the input line fields.

> 2. Click the [ASSIGN RECEIPT] button.  

Reviewer: der button wird bei mir nicht angezeigt. Muss da eine bestimmte Einstellung vorgenommen werden, damit das angezeigt wird? Ich habe allerdings auch in der Sandbox Dokumente nicht installiert - ist das eine Voraussetzung? Muss dann auf jeden Fall mit in die Prerequisites, da das nicht als allgemeine Voraussetzung im Kapitel General drin ist.. 
Ggf. nochmal Infos dazu einholen.

The *Attach Document* window is displayed.

  ![Attach document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachDocument.png "[Attach document]")

> 3. Select the document to be assigned to the booking from the *New documents* folder.

Reviewer: Lieber in zwei Schritten? 
1. Click the *New documents* folder in the side bar. -> All unassigned documents are displayed
2. Select the document to be assigned in the list of documents.
oder so ähnlich...

  ![Select document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachDocument01.png "[Select document]")

4. Click the [SELECT DOCUMENT] button.  

> The selected document is now assigned to the booking. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) will be then displayed next to the booking in the bookings list.

Reviewer: ```suggestion
The selected document has been assigned to the booking. The ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) is displayed next to the booking in the bookings list.
```

  ![Receipt assigned](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/ReceiptAssigned.png "[Receipt assigned]")

>   > [Info] When clicking the ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon, the corresponding receipt is displayed.

Reviewer: ```suggestion
  > [Info] Click the ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon to display the corresponding receipt.
```

5. If the receipt assignment needs to be undone due to a mistake, click the [CANCEL ASSIGNMENT] button.

>   ![Attached document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachedDocument.png "[Attached document]")

Reviewer: Kein eigener Schritt - ist ja nicht Bestandteil der Procedure, das Assignment wieder zu entfernen. Entweder eigene Procedure zum Löschen eines Assignments oder Einschließen in vorangehende Info

## Book a receipt

> The *BOOK RECEIPTS* tab allows to assign manually external receipts uploaded in the system that have not been registered yet in the *Accounting* module. Suppliers' invoices can be uploaded in PDF format in the *Documents* module, which will be then recognized by the system and displayed in the *BOOK RECEIPTS* tab.

Reviewer: ```suggestion
External receipts uploaded to the system and not yet registered can be assigned manually in the *Accounting* module. For example, suppliers' invoices can be uploaded as a PDF in the *Documents* module, which will be then recognized by the system and displayed in the *BOOK RECEIPTS* tab.
```

#### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).

> - A receipt is uploaded in PDF format in the *Documents* module.

Reviewer: ```suggestion
- A receipt is uploaded as a PDF in the *Documents* module.
```
ist PDF das einzige unterstützte Format?

#### Procedure

*Accounting > Select fiscal year > Book > Tab BOOK RECEIPTS*

![New receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NewDocument.png "[New receipt]")

>   > [Info] The external receipts uploaded in the *New documents* folder in the *Documents* module are displayed here. If there are no receipts to be booked, the message *No documents left to be registered* is displayed when clicking the *BOOK RECEIPTS* tab.
> 
>   ![No receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NoDocument.png "[No receipt]")

Reviewer: ```suggestion
  > [Info] All unassigned documents uploaded in the *New documents* folder in the *Documents* module are displayed in the *BOOK RECEIPTS* tab. If there are no unassigned documents, the notice *No documents left to be registered* is displayed in the *BOOK RECEIPTS* tab.
```
ohne Bild

> 

Reviewer: Muss der erste Schritt nicht der sein, das richtige Dokument auszuwählen?

> 1. Select the appropriate currency in the *Currency code* field, if necessary.

Reviewer: ```suggestion
1. If necessary, click the *Currency code* drop-down list and select the appropriate currency for the posting. All available currencies are displayed in the list. By default, the base currency is preselected.
```

  > [Info] Currency exchange rate can also be adjusted if needed in the *Rate* field.

2. Enter the total invoice value in the *Turnover* field.

3. Enter an account number in the *Contra Account* field.

4. Enter the date for the manual booking in the *Date* field. Date format must be DDMMYY or DD.MM.YY.

5. Enter the creditor (supplier) account number in the *Account* field.

  > [Info] Note that the fields *Contra Account*, *Date*,  *Account* must be filled out to be able to book a receipt.

6. Enter any other relevant booking information in the corresponding fields.

>   > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent bookings taking this field as reference.

Reviewer: vgl. manual booking

> 

Reviewer: vgl. manual booking

>   ![Booking a receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NewDocument01.png "[Booking a receipt]")

Reviewer: vgl. manual booking

7. Click the [BOOK] button.  

> A message above the entry line confirms that the receipt has been linked to the booking. A new booking is displayed in the bookings list in the *BOOKING* tab. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) is displayed next to the booking in the bookings list.

Reviewer: ```suggestion
The manual posting is created and linked to the selected receipt. The notice "Document assigned." is displayed above the input line. The new posting is displayed in the postings list in the *POSTINGS* tab. The ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) is displayed next to the posting in the postings list.
```

  ![Receipt linked](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/DocumentLinked01.png "[Receipt linked]")

>   > [Info] If there are further unregistered external receipts, they will be displayed in the *BOOK RECEIPTS* tab. Otherwise, the message "No documents left to be registered" is displayed.

Reviewer: ```suggestion
  > [Info] If there are further unregistered documents, they will be displayed in the *BOOK RECEIPTS* tab. Otherwise, the notice "No documents left to be registered" is displayed.
```

## Book a supplier receipt

> The *Supplier receipts* menu entry of the *EDIT* context menu in the *BOOKINGS* is an enhanced function to book specifically supplier receipts.

Reviewer: ```suggestion
In addition to the posting process and assignment of general receipts, there is an enhanced function to specifically post supplier receipts.
```

> Suppliers' invoices can be uploaded in PDF format in the *New supplier receipts* folder in the *Documents* module, which will be then recognized by the system and displayed in the *Acquire* tab in the *Supplier receipts* window. Once they are booked, they are moved automatically into the *Acquired documents* folder.

Reviewer: ```suggestion
Suppliers' invoices can be uploaded in PDF format in the *New supplier receipts* folder in the *Documents* module, which will be then recognized by the system and displayed in the *Acquire* tab in the *Supplier receipts* window. Once they are booked, they are moved automatically into the *Acquired documents* folder.
```
```suggestion
Suppliers' invoices can be uploaded as a PDF in the *New supplier receipts* folder in the *Documents* module, which will be recognized by the system and displayed in the *Acquire* tab in the *Supplier receipts* window. Once the supplier receipts are posted, they are moved automatically into the *Acquired documents* folder.
```

 Moreover, this function interacts with the *Purchasing* and *Payment processing* modules, and allows to check supplier receipts before booking them in the system and to automatize payment conditions.

[comment]: <> (ACQUIRE probably "enter" -ERFASSEN- or "park" -vorerfassen", SAP Terminology)

#### Prerequisites

> - The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
> - A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
> - The appropriate plug-in is installed (Checking of supplier receipts + payment proposal list).
> - The corresponding folder is created in the *Documents* module.
> - Supplier receipts are uploaded in the *New supplier receipts* folder.

Reviewer: ```suggestion
- A fiscal year has been selected, see [Select fiscal year](./01_SelectFiscalYear.md).
- The *Supplier receipts + payment proposal list* plug-in has been installed.
- At least one supplier receipt has been uploaded in the *New supplier receipts* folder.
```
Was meinst du mit dem Punkt "The corresponding folder is created in the *Documents* module."? Der New supplier receipts Ordner wird automatisch erstellt. Welcher Ordner wird sonst noch benötigt?

Author: Ich habe den Ordner selber erstellen müssen. Es war bei mir nicht vorhanden. Wann ist es automatisch erstellt?

#### Procedure

> *Accounting > Select fiscal year > Book > Tab BOOKINGS > Context menu EDIT > Menu entry Supplier receipts*
> 
> ![Supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceipts.png "[Supplier receipts]")
> 
> 1.  Click the *Supplier receipt* menu entry.  
> The *Supplier receipts* window is displayed. The external receipts uploaded in the *New supplier receipts* folder in the *Documents* module are displayed in the *Acquire* tab.

Reviewer: ```suggestion
*Accounting > Select fiscal year > Book > Tab BOOKINGS > Context menu EDIT > Menu entry Supplier receipts*

![Supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceipts.png "[Supplier receipts]")

1. Click the [EDIT] button in the header above the postings list.
A context menu is displayed.

2. Click the *Supplier receipts* menu entry.  
The *Supplier receipts* window is displayed. The external receipts uploaded in the *New supplier receipts* folder in the *Documents* module are displayed in the *Acquire* tab.
```

    ![Acquire supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsAcquire.png "[Acquire supplier receipts]")

    > [Info] If there are no receipts to be booked, the message *No documents left to be registered* is displayed.

>   [comment]: <> (They do not appear in Acquire if located in New supplier receipts folder, but in the New documents. Check why/if correct/if it is a bug?)

Reviewer: Ich bekomme nie Dokumente angezeigt, obwohl ich sowohl im Neue Dokumente Ordner, als auch im Neue Lieferantenbelege Ordner Dokumente liegen habe. Was mache ich falsch?

Author: Das war bei mir genauso. Heute hat es aber funktioniert, nachdem ich den Supplier receipts Ordner erstellt habe. Ich muss evtl. Flo oder Nseim nochmals fragen, aber ich meine, so sollte es funktioniert, wie jetzt beschrieben. Evtl. in die Einstellungen des Documents Moduls gucken, ob die Pfade stimmen?

2. Enter the total invoice value in the *Turnover* field.

  > [Info] Currency exchange rate can also be adjusted if needed in the *Exchange rate* field.

3. Enter the corresponding booking details in the *ContraAcct* and *Supplier* fields.

4. Enter the date for the manual booking in the *Date* field. Date format must be DDMMYY or DD.MM.YY.

5. Enter a description account number in the *Posting text* field.

  > [Info] Note that the fields *ContraAcct*, *Date*,  *Supplier* and *Posting text* must be filled out to be able to acquire a receipt.

6. Enter any other relevant booking information in the corresponding fields.

>   > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Reference1* field, as the system identifies and groups together any subsequent bookings taking this field as reference.

Reviewer: vgl manual booking

> 7. Select the appropriate payment target for the receipt in the *Payment target* drop-down list.

Reviewer: ```suggestion
7. Click the *Payment target* drop-down list and select the appropriate payment target for the receipt.
```
+ was wird angezeigt in der dropdown-liste

  ![Acquire supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsAcquire02.png "[Acquire supplier receipts]")

> 8. A split booking can also be performed with the [SPLIT] button, if necessary. For detailed information, see [Split a booking](./09_SplitBooking.md).

Reviewer: kein Schritt, sondern Info

9. Click the [ACQUIRE] button.  

> The supplier receipt is acquired and displayed in the *Overview* tab.  

Reviewer: ```suggestion
The supplier receipt has been acquired and is displayed in the *Overview* tab.  
```

  ![Overview](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsOverview.png "[Overview]")

> 10. The acquired receipts can be searched for using the *QUICK SEARCH* or the *ACTIVE FILTER* context menus.

Reviewer: ```suggestion
```
Überflüssig - warum sollte ich an dieser Stelle danach suchen?

>   > [Info] To be able to enter the supplier receipt in the system, they need to be released in the *Purchasing* module first. For detailed information, see [Check supplier receipts](#to-be-completed). Once they have been released in the *Purchasing* module, the status changes to **released** (colour yellow-green) in the *Status* column.

Reviewer: Verstehe ich nicht ganz - muss ich noch was im Einkaufs-Modul machen? Dann muss das als Schritt beschrieben werden. ..

11. Click the *Financial accounting takeover* tab and select the receipt to be entered in the system.

>   > [Info] When selecting a receipt, the receipt data are displayed in the entry line. Some changes can still be made here before takeover and saved with the [SAVE] button.

Reviewer: Grundsätzliche Terminologie: entry line vs. input line -> entscheide dich für eins und kontrolliere das nochmal in allen Dokumenten.

>  Double check all booking details and, if no further changes are needed, click the [TAKE OVER SELECTED ONES] button.   
>  The receipt is now registered in the *Accounting* module. The booking is displayed in the bookings list. An open item for the corresponding supplier (creditor) is created.

Reviewer: ```suggestion
 Double check all posting details and, if no further changes are needed, click the [TAKE OVER SELECTED ONES] button.   
 The receipt has been registered in the *Accounting* module. The posting is displayed in the postings list. An open item for the corresponding supplier (creditor) has been created.
```
Kann es sein, dass hier der Schritt (die Zahl) fehlt? Oder soll das ne Info sein?

[!!Documents](../../RetailSuiteDocorg/Overview/01_General.md)  
[!!Purchasing](../../RetailSuiteSourcing/Overview/01_General.md)   
[!!Payment processing](../../RetailSuiteBanking/Overview/01_General.md)  
[!!User Interface Postings](../UserInterface/01a_Bookings.md)  
[!!User Interface Post receipts](../UserInterface/01f_BookReceipts.md)  
[!!User Interface Supplier receipts](../UserInterface/01_Book.md#supplier-receipts)  
[!!User Interface Debtors/creditors](../UserInterface/02a_DebtorsCreditors.md)    

# Manage receipts

> Receipts are used to document all transactions and, therefore, play an indispensable role for a correct accounting. There are many different types of receipts, such as invoice, order confirmation, or credit note.

Reviewer 2: ```suggestion
Receipts are used to document all transactions and, therefore, play an indispensable role for a correct accounting. Every posting needs to have a receipt as a basis. There are many different types of receipts, such as invoice, credit note, cash voucher or rebooking document. 
```

2. In the input line under the document, click the *Currency code* drop-down list and select the appropriate currency for the posting. All currencies configured in the system are displayed in the list. By default, the base currency is preselected.

  > [Info] Currency exchange rate can also be adjusted if needed in the *Rate* field.

3. Enter the posting amount in the *Turnover* field. This field is mandatory.

  > [Info] The fields *Turnover*, *Contra account*, *Date* and *Account* must be filled to create a manual posting.

4. Enter an account number in the *Contra account* field.

  > [Info] After you have entered at least two numbers in the field, a list is displayed below the field with all accounts matching the entry. You can also select the corresponding account in the list.

5. Enter a reference number, usually the invoice number, in the *Receipt 1* field.

 > [Info] Even though it is not mandatory, it is highly recommended to fill in the *Receipt 1* field, as it is used as a reference to identify and assign any subsequent postings, for example for an invoice settlement or a credit note in case of refund.

> 

Reviewer 2: see comment in chapter 4

8. If desired, enter any other relevant manual posting information in the corresponding fields.

9. Click the [POST] button.  
The manual posting is created and linked to the selected receipt. The notice "Document linked. Posted." is displayed above the input line. The new posting is displayed in the postings list in the *POSTINGS* tab. The ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) is displayed next to the posting in the postings list.

  > [Info] If there are further unregistered documents, they will be displayed in the *POST RECEIPTS* tab. Otherwise, the notice "No documents left to be registered" is displayed.

## Post a supplier receipt

In addition to the posting process and assignment of general receipts, there is an enhanced function to specifically post supplier receipts.

Suppliers' invoices can be uploaded in the *New supplier receipts* folder in the *Documents* module, which will be recognized by the system and displayed in the *Acquire* tab in the *Supplier receipts* window. Once the supplier receipts are posted, they are moved automatically into the *Acquired documents* folder.

> Moreover, this function interacts with the *Purchasing* and *Payment processing* modules, and allows to check supplier receipts before posting them in the system and to automatize payment conditions.

Reviewer 2: ```suggestion
Moreover, this function interacts with the *Purchasing* and *Payment processing* modules, and allows to check supplier receipts before posting them in the system. For example can the Person who placed the order be responsible for approving the supplier receipt.

Funktion dient Zahlungsfreigabe, Zahlungsbedingungen werden im Lieferanten hinterlegt
```
